tests_root/wdio/package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.1.0",
   "description": "Automated end-to-end tests for Beep using WebdriverIO.",
   "scripts": {
+    "grid:up": "docker-compose up -d --scale chrome=1 --scale firefox=1",
+    "grid:down": "docker-compose down",
+    "test:desktop:grid": "yarn grid:up && sleep 10 && yarn test:desktop:grid:parallel && yarn grid:down",
```

Comment:
Can we stick to `npm` as that's what we use for our root scripts?

Reply:
I've been using `yarn` the last couple of months and I have it in my reflex 😃 
Addressed in commit 4965299. Thx!

Reply:
I'd rather use `yarn` myself, but for consistency's sake let's stick to `npm` thank you :)